### PR TITLE
Initramfs verify mtime

### PR DESCRIPTION
**Pull request by ddiss**

### CODE REVIEW

File: cut/initramfs_verify.sh [hunk 1]
```diff
@@ -9,19 +9,32 @@ _rt_require_dracut_args "$RAPIDO_DIR/autorun/initramfs_verify.sh" "$@"
...
+# append as a separate cpio archive.
+if ! grep -q "^# CONFIG_INITRAMFS_PRESERVE_MTIME" "$KERNEL_SRC/.config"; then
+	mkdir -p "${tmp_vdata}/fiod.mtime_chk/2"
+	touch --date=@1641548271 "${tmp_vdata}/fiod.mtime_chk"
+	touch --date=@1641548272 "${tmp_vdata}/fiod.mtime_chk/2"
+	echo -e "fiod.mtime_chk\nfiod.mtime_chk/2" \
+	  | cpio -o -H newc -D "$tmp_vdata"  >> "$DRACUT_OUT" \
+		|| _fail "failed to append mtime_chk archive"
+fi
+
```
Comment (luis-henrix) [inline cut/initramfs_verify.sh:39]: This doesn't look pretty but I guess there's not an easy way to work around it.  So... meh :-)

**Review by luis-henrix — APPROVED** [review]
This looks good.  I've gave it a try with different kernels, and it seems to work fine.

**Comment by ddiss** [comment]
Merged, thanks for the review Luis!

File: cut/initramfs_verify.sh (hunk 1 above)
Comment (ddiss) [reply cut/initramfs_verify.sh:39]: yeah, not ideal :-) . I figured it's too insignificant to bother fixing in Dracut for now. I might revisit it when I get to https://github.com/dracutdevs/dracut/issues/1662

**Pull request merged**